### ISSUE TITLE
Ramda applyJuxt function?

### ISSUE THREAD

**Issue by MattMS** [ramda/ramda#1824]
Is there a better way to do the following?

```
pipe(juxt([always('b'), prop('a'), identity]), apply(assoc))({a:1}) // = {a:1, b:1}
```

My thought was a function called `applyJuxt`, but I can't find anything like that.

**Comment by asaf-romano** [comment]
You're looking for ``converge``.
http://goo.gl/BqUwA6

We should cross-reference the two.

**Comment by MattMS** [comment]
That's perfect, thanks!
I really like the line `//≅ multiply( add(1, 2), subtract(1, 2) )` in the `converge` example, that highlighted it perfectly.

Yeah, even including `compose` in the "See also" list would be enough.
But ideally I'd have liked a sentence suggesting `converge` as the Array alternative.

**Comment by buzzdecafe** [comment]
closed by #1829
